acceptable/tests/test_main.py

```diff
@@ -472,15 +451,8 @@ def test_render_markdown_multiple_groups(self):
             iterator = main.render_markdown(metadata, args)
             output = OrderedDict((str(k), v) for k, v in iterator)
 
-            self.assertEqual(set([
-                    'en/group.md',
-                    'en/group2.md',
-                    'en/index.md',
-                    'en/metadata.yaml',
-                    'metadata.yaml',
-                ]),
-                set(output),
-            )
+            self.assertEqual({'en/group.md', 'en/group2.md', 'en/index.md', 'en/metadata.yaml', 'metadata.yaml'},
+                             set(output))
```

Comment:
Hm, I think this project would benefit from some [black](https://pypi.org/project/black/) magic :-)